app/src/test/java/org/oppia/android/app/testing/options/ReadingTextSizeFragmentTest.kt

```diff
@@ -114,20 +120,38 @@ class ReadingTextSizeFragmentTest {
   @Test
   fun testTextSize_changeTextSizeToLarge_changeConfiguration_checkTextSizeLargeIsSelected() {
     launch<ReadingTextSizeActivity>(createReadingTextSizeActivityIntent("Small")).use {
-      checkTextSize(SMALL_TEXT_SIZE)
-      updateTextSize(LARGE_TEXT_SIZE)
+      checkSelectedTextSize(SMALL_TEXT_SIZE)
+      clickOnTextSizeRecyclerViewItem(LARGE_TEXT_SIZE)
       rotateToLandscape()
-      checkTextSize(LARGE_TEXT_SIZE)
+      checkSelectedTextSize(LARGE_TEXT_SIZE)
+    }
+  }
+
+  @Test
+  fun testTextSize_checkTextSizeOfAllFourItems_textSizeMatchedCorrectly() {
+    launch<ReadingTextSizeActivity>(createReadingTextSizeActivityIntent("Small")).use {
+      checkTextSizeOfRecyclerViewItem(
+        SMALL_TEXT_SIZE, defaultTextSizeInFloat * SMALL_TEXT_SIZE_SCALE
+      )
+      checkTextSizeOfRecyclerViewItem(
+        MEDIUM_TEXT_SIZE, defaultTextSizeInFloat * MEDIUM_TEXT_SIZE_SCALE
+      )
+      checkTextSizeOfRecyclerViewItem(
+        LARGE_TEXT_SIZE, defaultTextSizeInFloat * LARGE_TEXT_SIZE_SCALE
+      )
+      checkTextSizeOfRecyclerViewItem(
+        EXTRA_LARGE_TEXT_SIZE, defaultTextSizeInFloat * EXTRA_LARGE_TEXT_SIZE_SCALE
+      )
     }
   }
 
   @Test
   @Config(qualifiers = "sw600dp")
   @LooperMode(LooperMode.Mode.PAUSED)
```

Comment:
```suggestion
```

We can remove using this as this is already annotated in class.

Reply:
done